lms/resources/_js_config/__init__.py

```diff
@@ -456,6 +456,13 @@ def _hypothesis_client(self):
                 }
             ]
         }
+        if self._context.is_canvas:
```

Comment:
Why is this Canvas only? Also this could really do with a comment explaining what this and what it does.